src/components/TodoList/TodoInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onClickDetails: (todo: Todo) => void;
+  chosedTodo: Todo | null;
```

Comment:
I think it would be better if you compute the value to boolean in TodoList component when you are mapping and just pass as prop isTodoChoosed